content/400-reference/300-database-reference/01-database-features.mdx

```diff
@@ -95,17 +95,17 @@ Lock option (MySQL):
-| JSON support                      | ✔️         | **✔️**†              | ✔️    | No     | ✔️\*                                                                   | ✔️\*                                                                                        | ✔️\*           |
-| Fuzzy/Phrase full text search     | ✔️         | ✔️                   | ✔️    | No     | Not yet                                                                | Not yet                                                                                     | Not yet        |
-| Table inheritance                 | ✔️         | ✔️                   | No    | No     | Not yet                                                                | ✔️\*                                                                                        | Not yet        |
+| Feature                           | PostgreSQL | Microsoft SQL Server | MySQL | SQLite | CockroachDB | Prisma schema                                                                               | Prisma Client | Prisma Migrate |
+| :-------------------------------- | :--------- | :------------------- | :---- | :----- | :---------- | :------------------------------------------------------------------------------------------ | ------------- | -------------- |
+| Autoincrementing IDs              | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | [`autoincrement()`](/concepts/components/prisma-schema/data-model#defining-a-default-value) | ✔️            | ✔️             |
+| Arrays                            | ✔️         | No                   | No    | No     | ✔️          | [`[]`](/concepts/components/prisma-schema/data-model#type-modifiers)                        | ✔️\*          | ✔️\*           |
+| Enums                             | ✔️         | No                   | ✔️    | No     | ✔️          | [`enum`](/concepts/components/prisma-schema/data-model#defining-enums)                      | ✔️\*          | ✔️\*           |
+| Native database types             | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | ✔️                                                                                          | ✔️            | Not yet        |
+| SQL Views                         | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | Not yet                                                                                     | Not yet       | Not yet        |
+| Authorization and user management | ✔️         | ✔️                   | ✔️    | No     | ✔️          | Not yet                                                                                     | Not yet       | Not yet        |
+| JSON support                      | ✔️         | **✔️**†              | ✔️    | No     | Not yet     | ✔️\*                                                                                        | ✔️\*          | ✔️\*           |
```

Comment:
Json itself is supported, it is only Json arrays/lists that are not.

Reply:
(This can either become just a checkmark, or it could get a footnote linking to the limitation table in the connector page)

Reply:
Thanks, have added a footnote